dodola/tests/test_services.py

```diff
@@ -582,48 +582,22 @@ def test_aiqpd_train(tmpdir, monkeypatch):
     )
 
     # remove leap days
-    temp_slice = convert_calendar(ds["air"], target="noleap")
+    ref_fine = convert_calendar(ref_fine["temperature"], target="noleap")
```

Comment:
I don't think you need this line if you use `xr.cftime_range`?

Reply:
Yep you're right